precompiles/bech32/Bech32.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
Update license

Reply:
To MIT license or ? Most other unique extension have this license.